Gifski/Gifski.swift

```diff
@@ -96,8 +96,9 @@ final class Gifski {
 			width: UInt32(conversion.dimensions?.width ?? 0),
 			height: UInt32(conversion.dimensions?.height ?? 0),
 			quality: UInt8(conversion.quality * 100),
+			once: false,
```

Comment:
@kornelski I assume `once` is ignored if `loop_count` is specified?

Reply:
Oops. No. This header got out of sync. Update to a newer commit.